components/base/IconImage.tsx

```diff
@@ -6,14 +6,14 @@ import { ImageData } from "../../data/domain/imageData"
 
 const BASE_URL = "https://cdn2.idleonefficiency.com/images"
 
-const IconImage = ({ data, scale = 1, style }: { data: ImageData, scale?: number, style?: React.CSSProperties }) => {
+export const IconImage = ({ data, scale = 1, style }: { data: ImageData, scale?: number, style?: React.CSSProperties }) => {
```

Comment:
There's already an `export default IconImage` at the bottom, so this isn't required. Either that or keep this and remove the other, I don't really know what's the difference haha

```suggestion
const IconImage = ({ data, scale = 1, style }: { data: ImageData, scale?: number, style?: React.CSSProperties }) => {
```

Reply:
I removed the export and kept the export default

The difference between having a default export and having two "basic" export is that if I keep the default I don't need to go back to every page using the IconImage to replace 
`import IconImage from '../../components/base/IconImage';`
by
`import { IconImage } from '../../components/base/IconImage';`

But by having a default one, on the rare pages that use the AdaptativeIconImage the import line will look like 
`import IconImage, { AdaptativeIconImage } from '../../components/base/IconImage';`
instead of 
`import { IconImage, AdaptativeIconImage } from '../../components/base/IconImage';`
when there was both default and "basic" export for IconImage

But it's pretty insignificant, and I think we'll very rarely have to use the AdaptativeIconImage, so it's better to not having duplicated export